deploy.md

```diff
@@ -68,6 +68,26 @@ rake db:create:all
 rake db:migrate 
 ```
 
+## Email configuration
+
+The Dashboard uses email to verify new accounts and to reset passwords.  In order to do this, it must be configured to know what email server to use.  This is configured in the file `config/environments/production.rb`, in the `config.action_mailer.smtp_settings` key:
+
+```
+config.action_mailer.smtp_settings = {
+    address: 'smtp.mandrillapp.com',
```

Comment:
Probably we should switch this address out to an envar. It seems likely that they will not have easy access to mandrillapp.